Mathlib/Order/CompleteLattice/SetLike.lean

```diff
@@ -0,0 +1,54 @@
+/-
+Copyright (c) 2024 Sven Manthe. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sven Manthe
+-/
+import Mathlib.Order.CompleteSublattice
```

Comment:
Is there a reason this is not in the `CompleteSublattice` file? Do you expect more material to join this file?

Reply:
The main reason is that I expect that most users of `CompleteSublattice` don't need this file.

Reply:
Hm. I see where you're coming from, but I'm not sure if this is a common argument... I'll wait for other reviewers to chime in.